system-x/services/ldap/src/main/java/software/tnb/ldap/validation/LdapValidation.java

```diff
@@ -0,0 +1,64 @@
+package software.tnb.ldap.validation;
+
+import software.tnb.common.validation.Validation;
+import software.tnb.ldap.account.LdapAccount;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.unboundid.ldap.sdk.LDAPConnection;
+import com.unboundid.ldap.sdk.LDAPException;
+import com.unboundid.ldap.sdk.SearchRequest;
+import com.unboundid.ldap.sdk.SearchResult;
+import com.unboundid.ldap.sdk.SearchResultEntry;
+import com.unboundid.ldap.sdk.SearchScope;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LdapValidation implements Validation {
+
+    private static final Logger LOG = LoggerFactory.getLogger(LdapValidation.class);
+    private LdapAccount ldapAccount;
+    private LDAPConnection ldapConnection;
+    private String ldapUrl;
+
+    public LdapValidation(LDAPConnection ldapConnection, LdapAccount ldapAccount, String ldapUrl) {
+        this.ldapConnection = ldapConnection;
+        this.ldapAccount = ldapAccount;
+        this.ldapUrl = ldapUrl;
+    }
+
+    public List<String> getEntries(String objectClass) {
```

Comment:
maybe to make the method a bit more universal, what about changing it to `public List<SearchResultEntry> getEntries(String filter)`?

Reply:
Done